model/mac-command.h

```diff
@@ -213,7 +219,15 @@ class LinkAdrReq : public MacCommand
 {
   public:
     LinkAdrReq();
-
+    /**
+     * Constructor with given fields.
+     *
+     * \param dataRate The DataRate field to set.
+     * \param txPower The TXPower field to set.
```

Comment:
unit of measure?

Reply:
Not here, these are serializable fields of the header for configurations specified in the LoRaWAN protocol. 
They will translate into modulation settings with physical units of measure for the PHY layer of end devices, but here the values (few bits) only represent them indirectly.

Reply:
Got it, How about specifying that this is a serialized value in the doc and basically what adding what you wrote here in the above comment?